src/main/java/seedu/address/logic/commands/TagCommand.java

```diff
@@ -75,10 +80,39 @@ public CommandResult addPersonTags(Model model) throws CommandException {
         personToEdit.setCommonModules(userModuleTags);
 
         model.updateObservablePersonList();
+
+        //
+        // need to take the set of module tags in this command
+        // loop through and then put into the
+        for (ModuleTag tag : moduleTags) {
+            String day = tag.getDay();
+            String start = tag.getStartTime();
+            String end = tag.getEndTime();
+            if (day == null || start == null || end == null) {
+                continue;
+            }
+
+            Module mod = new Module(tag.toString());
```

Comment:
Is there another more direct way of calling the tag value? Using the raw string can go wrong if we want to change that. :)